### PR TITLE
feat: less verbose output, new features

### LINKED ISSUE #19: make debug output conditional
like unspents, balances etc. Print to stdout only things necessary to monitor the progress

### PR DESCRIPTION
**Pull request by troggy**
- make verbose output conditional with "debug" package. Run with `DEBUG=* yarn test` for the verbose output
- use latest leap-provider features: `helpers.getProof`, `getUnspent`, `getPeriodByBlockHeight`, `getColors`

Resolves #19

### CODE REVIEW

File: src/nodeClient.js [hunk 1]
```diff
@@ -1,71 +1,55 @@
...
+      const blockNumber = await this.getBlockNumber();
 
       if (blockNumber > currentBlock) {
         break;
       }
 
-      let c = (await this.provider.send('plasma_getColors', [true, false]));
-      c = c.concat((await this.provider.send('plasma_getColors', [false, true])));
-      c = c.concat((await this.provider.send('plasma_getColors', [false, false])));
+      let c = Object.values(await this.getColors()).reduce((s, a) => s.concat(a), []);
```
Comment (sunify) [inline src/nodeClient.js]: `Object.values(await this.getColors()).flat()`

Comment (troggy) [reply src/nodeClient.js]: `TypeError: Object.values(...).flat is not a function` 🤔

Comment (troggy) [reply src/nodeClient.js]: ah.. not supported in node.js LTS yet

**Comment by troggy** [comment]
@eshavkun how is review going?

**Pull request merged**